src/api/v4/leaflet/layer-group.js

```diff
@@ -84,4 +92,23 @@ LayerGroup.prototype._triggerLayerFeatureEvent = function (eventName, internalEv
   }
 };
 
+function anyLayerHovered (map) {
```

Comment:
`map` is a bit misleading here... Perhaps we can just use a regular method and use `this._hoveredLayers` instead of passing `this._hoveredLayers` to a function.